localisation/00_zuwc_united_localisation.csv

```diff
@@ -2882,19 +2882,19 @@ summon_gorak_tul_desc;With our hold over Drustvar secure, we are now able to sum
 summon_gorak_tul_tooltip;�Y[From.GetBestName]�! summons the warband of Gorak Tul.;;�Y [From.GetBestName]�! beschw�rt die Krieger von Gorak Tul.;;;;;;;;;;;x
 heartsbane_coven_formed;The Heartsbane Coven has been formed;;Der Herzbannzirkel wurde gegr�ndet;;;;;;;;;;;x
 drust_opinion;[drust.GetName] Opinion;;[drust.GetName] Meinung;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+laid_eggs;Laid Eggs;;;;;;;;;;;;;x
+laid_eggs_desc;This character gets these bonuses and maluses per born child due to their government type.;;;;;;;;;;;;;x
+GetPeasantsType_workers;workers;;;;;;;;;;;;;x
+GetPeasantType_peon;peon;;;;;;;;;;;;;x
+GetPeasantType_acolyte;acolyte;;;;;;;;;;;;;x
+GetPeasantsType_peasant;peasant;;;;;;;;;;;;;x
+GetPeasantsType_worker;worker;;;;;;;;;;;;;x
+GetCattlesType_pigs;pigs;;;;;;;;;;;;;x
+GetCattlesType_livestock;livestock;;;;;;;;;;;;;x
+song_of_swarm_cooldown;Recent Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm_modifier;Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm;Song of Swarm;;;;;;;;;;;;;x
+song_of_swarm_desc;You sing to the swarm, giving them purpose to fight, your words filling their minds. Your desires are their desires, and they don't hesitate to obey.;;;;;;;;;;;;;x
```

Comment:
```suggestion
song_of_swarm_desc;You sing to the swarm, giving them the desire to fight, your words filling their minds. Your will is their will, and they don't hesitate to obey.;;;;;;;;;;;;;x
```

Reply:
Done in 21eff48138bfc52650c4c7c15c6ad1d6566bf162.